### DISCUSSION TITLE
Add Link to Push to Deploy Settings in Logs

### DISCUSSION THREAD

**Discussion by mvogttech** [meteor/meteor#11816] [Feature Requests]
After trying to convert my application to the new Push to Deploy feature, I noticed that the build kept failing. I tried to increase the hosted container size instead of the build container size.

`Error trying to start deploy - out of Memory in the build worker, try to increase the total available RAM in your app settings.`

I would append this line with a link to reconfigure the total available RAM.

**Comment by saksholm** [comment]
We got the same error. Multiple try and errors, I contacted support and we found something unusual. If your building is taking 1 hour, it's basically timeout and gives you that as a generic error. Build itself is working well, the root problem is in the upload part of deploy which takes a too long time. Of course, there is a chance that you facing another issue. Try to enable debug variables to see what's going on.

@renanccastro is already investigating this.

**Comment by renanccastro** [comment]
Good feedback, thanks @mvogttech . We will be implementing this as soon as possible!

Also, talking about the issue you @saksholm got, we increased the timeout for all users too, and the time to upload is being worked on as we speak. We will be changing the upload step to use custom go services that we have internally in a way to drastically cut this time.

**Reply by renanccastro** [reply]
btw @saksholm seems that was related to our EBS disk instance. We've changed to nvme ssds and things have improved a LOT, so I do consider this solved.
